addons/pos_sale/models/sale_order.py

```diff
@@ -26,6 +27,20 @@ def action_view_pos_order(self):
             'domain': [('id', 'in', linked_orders.ids)],
         }
 
+    @api.depends('invoice_ids.amount_total_signed', 'invoice_ids.state', 'pos_order_line_ids.price_subtotal_incl')
```

Comment:
so depends should depend on order_line (I don't think we edit down_payment_lines, so no need of order_line.amount_total)